test/node/parsing/WebGL/6

```diff
@@ -0,0 +1,552 @@
+typedef ([AllowShared] Uint32Array or sequence<GLuint>) Uint32List;
```

Comment:
I think the approach taken so far has been to extend the IDL parser for each source of IDL content if something is encountered that doesn't follow the spec.